metis/lib/models/file.rb

```diff
@@ -216,11 +216,15 @@ def unprotect!
       update(read_only: false)
     end
 
-    def rename!(new_folder, new_file_name)
-      update(
+    def rename!(new_folder, new_file_name, user=nil)
+      new_params = {
         file_name: new_file_name,
         folder_id: new_folder ? new_folder.id : nil
-      )
+      }
+
+      new_params[:author] = Metis::File.author(user) if user
```

Comment:
if !user, author is empty?

Reply:
If `!user`, `author` gets set to the default of `metis|Metis` (which is the current behavior you see on rename).